fastlane/Fastfile

```diff
@@ -69,14 +69,22 @@ lane :release do
         skip_upload_screenshots: true
     )
 
-
     sentry_create_release(
         org_slug: 'openfoodfacts',
         project_slug: 'openfoodfacts-android',
         version: PACKAGE_NAME + '@' + versionName + '+' + versionCode,
         finalize: false # Release will be finalized by the "finalize_sentry" lane, when the version is in production
     )
 
+    FLAVOR = SharedValues::GRADLE_FLAVOR + SharedValues::GRADLE_BUILD_TYPE
+    sentry_upload_proguard(
+      auth_token: ENV['SENTRY_AUTH_TOKEN'],
```

Comment:
@teolemon need your help, don't know how to inject token from CI

Reply:
I think there is no need to pass the variable, it is set globally with the correct name, therefore will be picked up by sentry. You can juste remove the parameter (same as other call to the sentry plugin, they are done without the parameter)

Reply:
noice! removed